crates/forge/tests/integration/declare.rs

```diff
@@ -54,6 +54,7 @@ fn simple_declare() {
 
     let result = run(
         &test.path().unwrap(),
+        &String::from("src"),
```

Comment:
Out of curiosity, I guess we could add anything here since this arg is irrelevant in integration test, right?

Reply:
Exactly